packages/p2p/src/components/recommend-user/recommend-user.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+
+const RecommendUser = ({ is_buy_order_for_user, onClickNotRecommended, onClickRecommended }) => {
+    const [is_no_selected, setIsNoSelected] = React.useState(false);
+    const [is_yes_selected, setIsYesSelected] = React.useState(false);
+
+    const selectNo = () => {
```

Comment:
NIT
```suggestion
    const handleSelectNo = () => {
```
Similarily, `handleSelectYes`?

Reply:
Sure, why not.